test/spec/lifecycle-test.js

```diff
@@ -881,6 +881,24 @@ describe("Lifecycle tests", function () {
           });
         });
       });
+
+      it("Does not cause elements to move when resizing with preventCollision=true and no compaction (#1933)", () => {
```

Comment:
Worth having a test where you can resize it (there's some empty space), as well as where you can resize it 1 grid unit but, say, not 2, because an item is in the way.

Reply:
Sounds good, happy to add those cases as well.